content/misc/glossary/contents.lr

```diff
@@ -105,6 +105,14 @@ A daemon is a computer program that runs as a background process, rather than be
 
 A special-purpose [relay](#relay) that maintains a list of currently-running relays and periodically publishes a [consensus](#consensus) together with the other directory authorities.
 
+### DoH
+
+DNS over HTTPS (DoH) is an internet security protocol which communicates domain name server information in an encrypted way over HTTPS connections.
```

Comment:
DoH communicates domain name *system* information over HTTPS, not domain name *server* information.